supplier/models.py

```diff
@@ -13,31 +12,12 @@ class Supplier(User):
     Attributes:
         business_name: a string which holds the Supplier business name.
     """
+    supplier_account = models.OneToOneField(User, on_delete=models.CASCADE, primary_key=True)
     business_name = models.CharField(max_length=32,
                                      validators=[MinLengthValidator(6)],
                                      null=False,
                                      help_text="Business Name")
 
-    @staticmethod
-    def filter_by_field(field, field_value):
-        """Fetches list of Suppliers filtered with given field and given value.
-
-        Args:
-        field:
-            Supplier attribute.
-        field_value:
-            value which will filter the suppliers.
-
-        Returns:
-            returns a queryset containing the objects that match the specified filter conditions.
-
-        Raises:
-        FieldDoesNotExist error: if Supplier doesn't have the specified attribute.
-        """
-        if hasattr(Supplier, str(field)):
-            return Supplier.objects.filter(**{field: field_value})
-        raise FieldDoesNotExist
```

Comment:
why delete? the main feature this function give is to check whether an attribute exists